crowbar_framework/index/index.html

```diff
@@ -114,7 +116,10 @@
     </div>
 
     <script>
-      document.getElementById("dashboard").href="http://" + window.location.host + ":3000";
+      links_to_dashboard = document.getElementsByClassName("dashboard");
```

Comment:
The so called 'strict mode' in javascript raises exception on variables declared without `var` keyword. So while this works, it is better to write `var links_to_dashboard = `.